src/app/pages/BlockDetailPage/index.tsx

```diff
@@ -87,7 +88,7 @@ export const BlockDetailView: FC<{
 
           <dt>{t('common.hash')}</dt>
           <dd>
-            <CopyToClipboard value={`0x${block.hash}`} />
+            <CopyToClipboard value={block.hash} label={isMobile ? trimLongString(block.hash) : block.hash} />
```

Comment:
I have a pending PR that would teach  CopyToClipboard to do this automatically. I think we should discuss what should be best approach here. 
In any case, until that happens, this is fine.

Reply:
I will link PRs. related to https://github.com/oasisprotocol/explorer/pull/106